packages/grafana-ui/src/components/DateTimePickers/TimeRangePicker/TimeRangeList.tsx

```diff
@@ -82,9 +82,15 @@ const getStyles = () => ({
   }),
 });
 
-const getOptionsStyles = () => ({
+const getOptionsStyles = (theme: GrafanaTheme2) => ({
   grow: css({
     flexGrow: 1,
     alignItems: 'flex-start',
   }),
+  list: css({
+    [theme.breakpoints.up('lg')]: {
```

Comment:
These styles are also applied to the quick range list, which truncates it:
![Screenshot 2024-02-16 at 13 41 31](https://github.com/grafana/grafana/assets/8878045/d48a33e4-1c59-4431-b0dc-4311c9fba2f2)